src/sections/Careers/Careers-Programs-grid/index.js

```diff
@@ -57,7 +57,8 @@ const ProgramsGrid = ({ hide_path, sub_section }) => {
 
   return (
     <ProgramsPageWrapper>
-      <PageHeader title="Open Source Internship Programs" subtitle="Build Your Career at Layer5" path={path} />
```

Comment:
is this pageheader used elsewhere also, i guess yes. So why only here the subtitle having the css issues and not elsewhere?

Reply:
yes, the pageHeader is used in other components too, and the subtitle has css issues in those places as well. For example, the issue exists on this page as well: https://layer5.io/community/members

Reply:
@sudhanshutech instead of the fix I did for the current page, should I fix the issue for all pages by adding the h3 class to the page header subtitle component?